addons/website/static/tests/tours/edit_megamenu.js

```diff
@@ -92,4 +92,68 @@ tour.register('edit_megamenu', {
         run: function () {}, // it's a check
     },
 ]);
+tour.register('edit_megamenu_big_icons_subtitles', {
+    test: true,
+    url: '/?enable_editor=1',
+}, [
+    // Add a megamenu item to the top menu.
+    {
+        content: "Click on a menu item",
+        trigger: '#top_menu .nav-item a',
+    },
+    {
+        content: "Click on 'Link' to open Link Dialog",
+        trigger: '.o_edit_menu_popover a.js_edit_menu',
+    },
+    {
+        content: "Trigger the link dialog (click 'Add Mega Menu Item')",
+        extra_trigger: ".o_web_editor_dialog:visible",
+        trigger: '.modal-body a.js_add_menu[data-type="mega"]',
+    },
+    {
+        content: "Write a label for the new menu item",
+        extra_trigger: ".o_link_dialog",
+        trigger: '.o_link_dialog #o_link_dialog_label_input',
+        run: 'text Megaaaaa!',
+    },
+    {
+        content: "Save the new menu item",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
+        run: 'click',
+    },
+    {
+        content: "Save the changes to the menu",
+        trigger: '.modal-dialog .btn-primary span:contains("Save")',
+        run: 'click',
+    },
+    // Edit a menu item
+    wTourUtils.clickOnEdit(),
+    wTourUtils.clickOnExtraMenuItem({extra_trigger: '#oe_snippets.o_loaded'}),
+    toggleMegaMenu({extra_trigger: '#top_menu .nav-item a.o_mega_menu_toggle:contains("Megaaaaa!")'}),
+    {
+        content: "Select the first menu link of the first column",
+        trigger: '.s_mega_menu_odoo_menu .row > div:first-child .nav > :first-child',
+    },
+    wTourUtils.changeOption("MegaMenuLayout", "we-toggler"),
+    {
+        content: "Select Big Icons Subtitles mega menu",
+        trigger: '[data-select-label="Big Icons Subtitles"]',
+    },
+    wTourUtils.clickOnExtraMenuItem({extra_trigger: '#oe_snippets.o_loaded'}),
+    {
+        content: "Select the h4 of first menu link of the first column",
+        trigger: '.s_mega_menu_big_icons_subtitles .row > div:first-child .nav > :first-child .media-body > h4',
+    },
+    {
+        content: "Convert it to Bold",
+        trigger: '#oe_snippets #toolbar #bold',
+    },
+    ...wTourUtils.clickOnSave(),
+    toggleMegaMenu(),
+    {
+        content: "The menu item should only convert selected text to Heading 1.",
+        trigger: '.s_mega_menu_big_icons_subtitles .row > div:first-child .nav > :first-child .media-body > :last-child:not(:has(strong))',
```

Comment:
@qsm-odoo Does it seem ok to you to test with a specific template and its quite specific html content? Seems like something that's gonna be in the way for every little redesign and at the same time it's a tour so why not.

Reply:
@qsm-odoo Could you please have a look at the tour.

Reply:
@Zynton I did not see your ping sorry. A sub-comment in a review to which I am already indirectly subscribed as a code-owner, I'd say it's close to a 50% chance that I see it :grimacing: In general, you can do a ping in the main review comment, I'll understand I'll have to scroll through the sub comments... :pray: (Which I often do anyway to be sure but like I said 50% chance to catch it in that case, if I don't want my job to be just about "scrolling through emails" ^^).

For the test, I think it's ok to rely on the current template, it's not like we change mega menu design every 3 weeks. However, are you sure this should not be about making a lower-level test about "supporting links with complex structure inside"? In that case, would be easier to make a test that will not have to change at long-term? But as you wish :+1: